typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -468,12 +468,13 @@ export class EvmIsmModule extends HyperlaneModule<
 
       // initialize the fallback routing ISM
       logger.debug('Initializing fallback routing ISM ...');
-      await ism['initialize(address,uint32[],address[])'](
+      const tx = await ism['initialize(address,uint32[],address[])'](
```

Comment:
wonder if we should have some lint rule around always capturing the result of an await - tricky one to find